MobiFlight/MobiFlightLedModule.cs

```diff
@@ -65,6 +66,14 @@ public void Display(int subModule, String value, byte points, byte mask)
 
             var command = new SendCommand((int)MobiFlightModule.Command.SetModule);
 
+            var strValueParts = value.Split('.');
+            value = String.Join("", strValueParts);
```

Comment:
Instead of calling this `value` and then having to re-join the string later if there is a decimal, why not call this `valueWithoutDecimals`?

Reply:
Thinking about this even more, why even do this at all? Just pass `value` to `CalcuateCorrectPoints()` and when you need the string without decimals do `value.Replace('.', '')`;